### ISSUE TITLE
[need writting] series deep learning cơ bản

### ISSUE THREAD

**Issue by ducanh841988** [dlapplications/dlapplications.github.io#7]
Mọi người cùng thảo luận nhé. Nếu thấy mình viết được topic nào thì tự tạo task trên todo list.
Các bài tiếp theo series của Tuấn [deep learning cơ bản] mình đề xuất như sau: 
+ gradient decent
~~+ Giới thiệu các framework để làm thực hành~~
~~+ Chọn 1 cái framework và hướng dẫn sử dụng~~
+ perceptron and MLP lab  
+ Lý thuyết CNN
+ lab CNN
+ Lý thuyết RNN, LSTM. cái này có thể phân ra vài bài
+ Lab RNN, LSTM

**Comment by hitogen** [comment]
Sau bài MLP e sẽ định viết tiếp bài về loss function, gradient descent, backprob và sẽ có demo MNIST cho MLP xài tf hoặc pytorch

**Comment by hitogen** [comment]
Phần này mà rủ a Thuận vô viết cùng đc thì tốt :))

**Comment by gachiemchiep** [comment]
@ducanh841988 @hitogen 

Cai issue nay lon qua. nhet sang milestones (dl_basic) moi nhe.

**Comment by hitogen** [comment]
@gachiemchiep milestone nó có ý nghĩa như nào đại ca

**Comment by ducanh841988** [comment]
@hitogen : em viet tiep ve gradent dung ko?
Ban cua Thor dang viet ve CNN ah.
Neu duoc em lo luon phan lab cua MLP nhe. Anh se lo phan lab CNN.
RNN thi giao cho Thuan va em ko biet duoc ko?

**Comment by hitogen** [comment]
@ducanh841988 vâng e viết tiếp gradient và tất nhiên có lab cho MLP. RNN e viết cũng được ạ mà chắc hơi mất thời gian vì cái Back prob through time nó dài dòng.

**Comment by nxthuan512** [comment]
Trang này rất giống mấy cái mọi người định làm na. Code rất dễ hiểu và dễ phát triển thêm. 
https://machinelearningmastery.com/

**Comment by gachiemchiep** [comment]
@ducanh841988 

Tạo task control cho series này nhé. Như ở https://github.com/dlapplications/dlapplications.github.io/issues/19
